wgpu-hal/Cargo.toml

```diff
@@ -8,7 +8,7 @@ homepage.workspace = true
 repository.workspace = true
 keywords.workspace = true
 license.workspace = true
-rust-version.workspace = true
+rust-version = "1.60"
```

Comment:
~~This isn't true because of manifest inheritance.~~

Could you add a note why this is valid (cargo vendor) despite looking contradictory.